dist/_grid.scss

```diff
@@ -1,29 +1,43 @@
-@function breakpoint-value($key) {
-  @if map-has-key($breakpoints, $key) {
-    @return map-get($breakpoints, $key);
+@function breakpoint-value($breakpoint) {
+  @if map-has-key($breakpoints, $breakpoint) {
+    @return map-get($breakpoints, $breakpoint);
   }
 }
 
-@function shorthand-name($key) {
-  @if map-has-key($shorthands, $key) {
-    @return map-get($shorthands, $key);
+@function shorthand-name($shorthand) {
+  @if map-has-key($shorthands, $shorthand) {
+    @return map-get($shorthands, $shorthand);
   }
 }
 
-@function requires-media-query($key) {
-  @if (breakpoint-value($key) == 0) {
+@function requires-media-query($breakpoint) {
+  @if (breakpoint-value($breakpoint) == 0) {
     @return false;
   }
   @return true;
 }
 
-@function requires-extension($key) {
-  @if (requires-media-query($key)) {
-    @return '_#{$key}';
+@function breakpoint-extension($breakpoint) {
+  @if (requires-media-query($breakpoint)) {
+    @return '_#{$breakpoint}';
   }
   @return '';
 }
 
+$shorthands: (
+  2: 'two_across',
+	3: 'three_across',
+	4: 'four_across',
```

Comment:
Not sure why this is appearing as tabs. My 2nd commit clearly changes these to spaces. 🤔